tldr/hybrid_extractor.py

```
@@ -184,10 +195,11 @@ class HybridExtractor:
     LUA_EXTENSIONS = {".lua"}
     LUAU_EXTENSIONS = {".luau"}
     ELIXIR_EXTENSIONS = {".ex", ".exs"}
+    ZIG_EXTENSIONS: ClassVar[set[str]] = {".zig"}
 
-    def __init__(self):
+    def __init__(self) -> None:
         self._pygments_extractor = SignatureExtractor()
-        self._ts_parsers: dict[str, Any] = {}
+        self._ts_parsers: "dict[str, Parser]" = {}
```

Ruff (0.14.14)

Remove quotes from type annotation

Remove quotes

(UP037)